spec/descriptions/tagSyntheticResults.md

```diff
@@ -79,17 +101,17 @@ To narrow down the result set you have two options to search for a test.
 {
     "testId": ["tUmWgvzdo1Q1vpVRpzR5", "Pg0Q1UqHRd7OMysohVLd"],
     "//comment1": "Get test results from last 30 minutes (windowSize), data are aggregated every 10 minutes (granularity)",
-    "//comment2": "The granularity values for response_time and response_size must be the same"
+    "//comment2": "The granularity values for responseTime and responseSize must be the same"
     "metrics": [
      {
         "aggregation": "MEAN",
         "granularity": 600,    
-        "metric": "response_time"
+        "metric": "synthetic.metricsResponseTime"
```

Comment:
Both "response_time" and "synthetic.metricsResponseTime" work here. Same for the response_size and synthetic.metricsResponseSize.